LoopUI/Models/ChartAxisValueDoubleUnit.swift

```diff
@@ -1,31 +0,0 @@
-//
```

Comment:
Moved because it's needed by `GlucoseChart`, which is needed for the insulin model chart